js/gdm/authPrompt.js

```diff
@@ -583,6 +583,33 @@ var AuthPrompt = class {
         this.emit('cancelled');
     }
 
+    _getFileContents(filename) {
```

Comment:
This method doesn't access `this` anywhere. It's better to move it to `js/mist/fileUtils.js` as a standalone function.

Reply:
Done.